validphys2/src/validphys/replica_selector.py

```diff
@@ -25,12 +25,12 @@ def _fixup_new_replica(alphas_pdf: PDF, new_replica_file):
     to the header file.
     """
     AlphaS_MZ = alphas_pdf.AlphaS_MZ
-    AlphaS_Vals = alphas_pdf.AlphaS_Vals
+    AlphaS_vals = alphas_pdf.AlphaS_vals
```

Comment:
I guess this would crash and burn for all our existing sets? Maybe add a special case (with a comment)?